### PR TITLE
Add url params to gitignore.io

### PR DESCRIPTION
**Pull request by codeOfRobin**
Added URL Params so keywords can be populated on first load. On loading `templates.json`, it’ll now extract URLParams and filter out those params that exist in the templates. It’ll then “fill” them in the `select2` Search box.

Signed-off-by: Robin Malhotra <me@rmalhotra.com>

### CODE REVIEW

**Comment by joeblau** [comment]
This works beautifully.  Let's change the "keywords" url param to "templates" and then we can merge this in.  I'll also make a task to create a direct link to update the template in the the generated files.

File: Public/js/app.js [hunk 1]
```diff
@@ -9,7 +9,13 @@ $.ajax('/dropdown/templates.json').success(data => {
         theme: "bootstrap",
         multiple: true,
         data
-    });                                                                            
+    });
+
+    const urlParams = new URLSearchParams(window.location.search);
+    const preFilledSearchTerms = urlParams.get("keywords").split(",").map(val => val.toLowerCase())
```
Comment (joeblau) [inline Public/js/app.js]: Chanke "keywords" to "templates"

Comment (codeOfRobin) [reply Public/js/app.js]: Should be done now!

**Comment by joeblau** [comment]
Fixes #421

**Pull request merged**